src/CodeGen_Hexagon.cpp

```diff
@@ -493,6 +493,9 @@ void CodeGen_Hexagon::init_module() {
         // Absolute value:
         { IPICK(is_128B, Intrinsic::hexagon_V6_vabsh),   u16v1, "abs.vh", {i16v1} },
         { IPICK(is_128B, Intrinsic::hexagon_V6_vabsw),   u32v1, "abs.vw", {i32v1} },
+#if LLVM_VERSION >= 50
+        { IPICK(is_128B, Intrinsic::hexagon_V6_vabsb),   u8v1, "abs_v65.vb", {i8v1} },
```

Comment:
I don't think this should have a prefix different from the rest (none of the other >v60 instructions do).